src/components/alert/__definition__.js

```diff
@@ -21,7 +21,6 @@ let definition = new Definition('alert', Alert, {
     size: OptionsHelper.sizesFull
   },
   propValues: {
-    ariaRole: 'alertdialog',
```

Comment:
We should keep this so the demo site renders the `Alert` with the correct ARIA role.